src/aleph/vm/orchestrator/views/authentication.py

```diff
@@ -154,13 +157,17 @@ def get_signed_pubkey(request: web.Request) -> SignedPubKeyHeader:
         raise web.HTTPBadRequest(reason="Invalid X-SignedPubKey fields") from error
     except json.JSONDecodeError as error:
         raise web.HTTPBadRequest(reason="Invalid X-SignedPubKey format") from error
-    except ValueError as error:
-        if error.args == ("Token expired",):
-            raise web.HTTPUnauthorized(reason="Token expired") from error
-        elif error.args == ("Invalid signature",):
-            raise web.HTTPUnauthorized(reason="Invalid signature") from error
+    except ValueError as errors:
```

Comment:
because it contains a list of validation errors, which is why we loop on it. Don't know if it is somethings that changed in pydantic but the previous code was not working